scripts/windows-sign.js

```diff
@@ -0,0 +1,72 @@
+// This signing procedure only runs on windows binary builds to leverage remote signing in order to
+// fullfil new requirements around OV and IV code signing.
+// @see https://www.ssl.com/article/code-signing-key-storage-requirements-will-change-on-june-1-2023/ for more details.
+
+// Signing delegation happens inside the electron-builder.json, which can be seen in the configuration:
+// "sign": "./scripts/windows-sign.js"
+// @see https://www.electron.build/configuration/win#how-do-delegate-code-signing for configuration reference.
+
+// sampled from https://github.com/electron-userland/electron-builder/issues/6158#issuecomment-899798533
+const path = require('path')
+const fs = require('fs')
+const childProcess = require('child_process')
+
+const TEMP_DIR = path.join(__dirname, 'release', 'temp')
```

Comment:
```suggestion
import { tmpdir } from 'os'
const TEMP_DIR = path.join(tmpdir, 'release', 'tmp')
```

Reply:
addressed in [befe706](https://github.com/cypress-io/cypress/pull/28636/commits/befe706387452cb1337fb45da320b22ef826af0c)